ci-operator/step-registry/gather/loki/gather-loki-commands.sh

```diff
@@ -37,23 +54,12 @@ function getlogs() {
   mkdir -p "/tmp/loki-container-logs/${ns}/${pod_name}"
   target="/tmp/loki-container-logs/${ns}/${pod_name}/${container_name}_${container_pos}"
 
-  # TODO(jchaloup): get pod create timestamp from labelscl
-  # local lokits
-  # lokits=$(echo ${dataline} | jq -r 'select (.data!=null) | select (.data.result!=null) | select (.data.result[0].values!=null) | select (.data.result[0].values[0]!=null) | select (.data.result[0].values[0][0]!=null) | .data.result[0].values[0][0]')
-  # if [[ -n "${lokits}" ]]; then
-  #   container_date="$(date -u -d @${lokits:0:10} +'%Y-%m-%dT%H:%M:%S')"
-  #   echo "container_date: ${container_date}"
-  #   mkdir -p "/tmp/loki-container-logs/$ns/$pod_name/${container_date}"
-  #   target="/tmp/loki-container-logs/$ns/$pod_name/${container_date}/$container_name"_"$container_pos"
-  # fi
-
   # 2020-04-16T18:23:30+02:00 {} 2020-04-16T16:23:29.778263432+00:00 stderr F I0416 16:23:29.778201       1 sync.go:53] Synced up all machine-api-controller components
-  timeout 1m curl -s \
-    --data-urlencode "query={filename=\"${filename}\"}" \
-    --data-urlencode "limit=10000000" \
-    --data-urlencode "start=0" \
+  timeout 1m curl -G -s \
+    --header "Authorization: Bearer ${ACCESS_TOKEN}" \
+    --data-urlencode "query={filename=\"${filename}\", _id=\"${CLUSTER_ID}\"}" \
+    --data-urlencode "start=${START}" \
```

Comment:
Is the re-export per file name a must-have? Can we accomplish this w/o querying loki?

Reply:
I think it would be useful - its convenient when logs are also backed up to GCS, especially while Grafana is not yet setup to list cluster logs.

Any concerns in regards to that?

Reply:
I would like not to stress Loki on staging yet. Can we achieve this backup with other means? e.g. copying the logs from nodes directly to GCP in addition to loki?

Reply:
Oh yeath, I'll keep this PR in WIP until we move to prod instance, so the only clusters using stage instance would be rehearsals.

Nevertheless I'd like to keep "fetch logs for the containers and save in GCS" feature in CI. This seems to be convenient for developers

Reply:
Since taking to prod could take a while. Can we solve this w/o Loki for now? I would like to merge this and try on cluster logging jobs.

Reply:
That can be done, but we'd have to introduce a new step / chain / workflow for that

Reply:
Awesome, let's make GCP archiving a separate step then. Thank you! :rocket: